doc/user_guide/Custom_Interactivity.rst

```diff
@@ -2,4 +2,5 @@ Custom Interactivity
 ____________________
 
 .. notebook:: holoviews ../../examples/user_guide/12-Custom_Interactivity.ipynb
+    :skip_execute: True
```

Comment:
> I've added the print output to the notebook for now.

That is fine for now but isn't a long term solution.

```
1+1
# When run live, this cell's output should match the behavior of the GIF below
hv.DynamicMap(....)
```
[GIF in markdown here]

The comment can be anywhere in the cell you want to skip. The rst:

```rst
.. notebook:: holoviews ../../examples/user_guide/12-Custom_Interactivity.ipynb
    :skip_output: When run live, this cell's output should match the behavior of the GIF below
```

See the Live Data rst for the getting started guide...